src/Compatibility/Core/src/Android/AppCompat/Platform.cs

```diff
@@ -81,7 +80,7 @@ internal bool NavAnimationInProgress
 					return;
 				_navAnimationInProgress = value;
 				if (value)
-					MessagingCenter.Send(this, CloseContextActionsSignalName);
+					WeakReferenceMessenger.Default.Send(new CloseContextActionsMessage());
```

Comment:
Nit: this could be simplied through the extension:

```suggestion
					WeakReferenceMessenger.Default.Send<CloseContextActionsMessage>();
```